src/main/java/marquez/api/GetAllNamespacesResponse.java

```diff
@@ -0,0 +1,25 @@
+package marquez.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.List;
+import marquez.core.models.Namespace;
+
+public class GetAllNamespacesResponse {
+
+  List<GetNamespaceResponse> namespaces = new ArrayList<>();
+
+  public GetAllNamespacesResponse(@JsonProperty("namespaces") List<Namespace> namespaceList) {
```

Comment:
Very minor, but I think `ListNamespacesResponse` instead of `GetAllNamespacesResponse` translate more to the action of the caller.

Reply:
Sounds good. Less awkward-sounding this way.